src/main/java/seedu/address/ui/StatisticsDisplay.java

```diff
@@ -31,18 +40,13 @@
     @FXML private ImageView progressBarDaily;
     @FXML private Label medals;
 
-    public StatisticsDisplay(
-            String barChartTitleText,
-            int[][] data,
-            Path progressBarDailyFilepath,
-            String progressDailyText,
-            String medalsText) {
+    public StatisticsDisplay() {
         super(FXML);
-        this.barChartTitleText = barChartTitleText;
-        this.data = data;
-        this.progressBarDailyFilepath = progressBarDailyFilepath;
-        this.progressDailyText = progressDailyText;
-        this.medalsText = medalsText;
+        this.barChartTitleText = DEFAULT_BAR_CHART_TITLE;
+        this.data = null; // TODO *******************
```

Comment:
We definitely shouldn't read from JSON directly here, should be done from maybe ModelManager so that every time a command is executed, statistic is updated

Reply:
No worries, I have not set up that class yet! I will work on it once Hardy finished up data generation for statistics.json. Code here is meant only for "default" placeholders